container/ubuntu14_04/layers/gcc/gcc.yaml

```diff
@@ -5,3 +5,8 @@ commandTests:
   command: 'g++'
   args: ['--version']
   expectedOutput: ['g[+][+] [(]Ubuntu .*']
+
+metadataTest:
+  env:
+    - key: 'GCOV'
```

Comment:
I think gcov can be set to something useful with GCC, but not with clang, you can probably check in the container just by trying to find files with name *gcov*